.github/workflows/build-app.yml

```diff
@@ -5,5 +5,11 @@ jobs:
   build-woof-target:
     runs-on: macos-13
     steps:
-      - uses: actions/checkout@v3
-      - run: ./scripts/build-ios-simulator.sh
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: Set up Xcode
+        uses: maxim-lobanov/setup-xcode@v1
```

Comment:
Thanks for your comments it's a draft PR, I haven't requested a review yet, it will be not the part of this PR, because @iBYarchitect  helps me with fixing the problem that raised during the build phase on github action.

Reply:
Update the script with https://github.com/ios-course/ironfoudation-team-project/pull/188/commits/40d2aadf5231832ee2f5fe0fa862b3dcfe3dec32